tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -422,6 +431,14 @@ def _read_latest_config_files(self, run_path_pairs):
                 with tf.io.gfile.GFile(config_fpath, "r") as f:
                     file_content = f.read()
                 text_format.Merge(file_content, config)
+            elif tf.io.gfile.exists(default_config_fpath):
+                with tf.io.gfile.GFile(default_config_fpath, "r") as f:
```

Comment:
This looks like it would read the same file contents over and over, once for each run that doesn't have its own projector config, which seems unnecessary. Can we restructure so that we only read the file at most once?  This isn't such a big deal for local filesystems but is more of a downside for e.g. GCS log directories.